src/django/api/migrations/0001_initial.py

```diff
@@ -136,4 +137,8 @@ class Migration(migrations.Migration):
             name='created_from',
             field=models.OneToOneField(help_text='The original uploaded list item from which this facility was created.', on_delete=django.db.models.deletion.PROTECT, related_name='created_facility', to='api.FacilityListItem'),
         ),
+        migrations.AddIndex(
+            model_name='facilitylistitem',
+            index=models.Index(fields=['facility_list', 'row_index'], name='api_facilit_facilit_196927_idx'),
```

Comment:
Should this be given a friendlier name?

Reply:
Good thought. At first I though we don't need to bother, but the index name shows up in `EXPLAIN ANALYZE` output so I think it would be nice to call it something like `facility_list_item_facility_list_id_row_id`.

Reply:
Or better, for consistency, `api_facilitylistitem_facility_list_id_row_id`

Reply:
Unfortunately,

```
ValueError: ['Index names cannot be longer than 30 characters.']
```

This is [a Django limitation](https://docs.djangoproject.com/en/2.0/ref/models/indexes/#name), not a Postgres one, which explains the existence of the 46 character long `api_facilitylistitem_facility_list_id_ad3f851d`.

I'll use `api_fli_facility_list_row_idx` which is 29.